test/integration/auth_test.go

```diff
@@ -169,6 +169,14 @@ var aEndpoints string = `
 }
 `
 
+var deleteNow string = `
+{
+	"kind": "DeleteOptions",
+	"apiVersion": "v1beta1",
+	"gracePeriod": 0%s
```

Comment:
Extra data added to all requests.  Sprintf adds extra data if you send a value and there's no format, so the substitution, while empty, is necessary.

> On Mar 18, 2015, at 8:35 PM, Brian Grant <notifications@github.com> wrote:
> 
> In test/integration/auth_test.go:
> 
> > @@ -169,6 +169,14 @@ var aEndpoints string = `
> >  }
> >  `
> >  
> > +var deleteNow string = `
> > +{
> > +	"kind": "DeleteOptions",
> > +	"apiVersion": "v1beta1",
> > +	"gracePeriod": 0%s
> gracePeriodSeconds.
> 
> What's the %s for?
> 
> —
> Reply to this email directly or view it on GitHub.
>